django_downloadview/apache/response.py

```diff
@@ -7,9 +7,12 @@
 class XSendfileResponse(ProxiedDownloadResponse):
     "Delegates serving file to Apache via X-Sendfile header."
 
-    def __init__(self, file_path, content_type, basename=None, attachment=True):
+    def __init__(self, file_path, content_type, basename=None, attachment=True, headers=None):
         """Return a HttpResponse with headers for Apache X-Sendfile."""
-        super().__init__(content_type=content_type)
+        # content-type must be porvided only as keyword argument to response
```

Comment:
There is a small spelling error in the word and the same error is made in other files
```suggestion
        # content-type must be provided only as keyword argument to response
```

Reply:
Fixed, thank you.